### PR TITLE
Environments

### PR DESCRIPTION
**Pull request by jholdstock**
- Add a new page for regnet
- Make all environment pages good enough for production

Opened #13 for adding more detail to the mainnet page because it is a bit sparse at the moment.

### CODE REVIEW

File: docs/environments/testnet.md [hunk 1]
```diff
@@ -59,6 +51,13 @@ listed at the bottom of that page when you're done playing with the testnet.
...
 ---
 
+## Testnet Block Explorer
+
+A testnet instance of [dcrdata](https://github.com/decred/dcrdata) is available for public
+use at <https://testnet.dcrdata.org/>.
+
+---
+
 ## Testnet Voting Service Providers (VSP)
```
Comment (s-ben) [inline docs/environments/testnet.md]: Maybe provide a little more info on the list of testnet VSPs if possible? E.g. "Below is a list of VSPs that support testnet"

https://test-dcrpool.dittrex.com/ is returning a 502 error...

File: docs/environments/testnet.md [hunk 2]
```diff
@@ -1,40 +1,32 @@
...
 file for testnet3 use. Your mainnet blockchain and wallet files will remain
-untouched. Switching between the two is incredibly easy.
+untouched.
 
----
-
-## Decrediton
+### Decrediton
 
 You can switch Decrediton between mainnet and testnet in the Settings tab by
```
[the remark this answers is not in the document]
Comment (jholdstock) [reply docs/environments/testnet.md]: Hmm not sure about this one. As you say, I don't really think we have an established standard. Lets open an issue

[the remark this answers is not in the document]
Comment (jholdstock) [reply docs/environments/testnet.md]: #17

File: docs/environments/testnet.md (hunk 1 above)
Comment (jholdstock) [reply docs/environments/testnet.md]: Added a caveat:

> As these are test environments, they may experience periods of instability or downtime.

**Comment by jholdstock** [comment]
Thanks. I have implemented all suggestions and created a new issue for standardizing how we refer to gui elements (`code` or **bold**).

I think this is good to go now

**Pull request merged**